tests/sles4sap/robot_fw.pm

```diff
@@ -32,6 +33,7 @@ sub run {
     # Install the robot framework
     assert_script_run "unzip /robot/bin/robotframework-$robot_fw_version.zip";
     assert_script_run "cd robotframework-$robot_fw_version";
+    zypper_call "in $python_bin-setuptools" if (script_run "rpm -qa | grep $python_bin-setuptools");
```

Comment:
I think the conditional `if` is not needed at all because `zypper`returns 0 if the package is already installed.

Reply:
yes I know, same answer as above.